INSTALL.rst

```diff
@@ -44,7 +44,7 @@ Compilation on Windows
 
 Install `MSYS2 <https://www.msys2.org/>`_ and `Python <https://www.python.org/downloads/windows/>`_ on your system.
 Launch a `MINGW64 environment <https://www.msys2.org/docs/environments/>`_ and
-install the required packages for building cvc5:
+install the required packages for building cvc5 (you can install `ninja` instead of `make` if you prefer):
```

Comment:
This comment is incorrect. Ninja doesn't work properly in MSYS2 when building cvc5.